racksdb/web/ui/src/components/InfrastructureTable.vue

```diff
@@ -102,6 +168,26 @@ function getRackEquipments(rackName: string) {
       })
     )
   })
+
+  equipments.forEach((equipment) => {
```

Comment:
This introduces a side-effect in `getRacksEquipment()`. When you call this function, you don't expect it to also fill `racks`, `equipmentCategories`, `equipmentTypes` and `tags` global arrays.